crates/forge/tests/data/component_macros/Scarb.toml

```
@@ -1,12 +1,12 @@
 [package]
 name = "component_macros"
 version = "0.1.0"
-edition = "2023_01"
+edition = "2024_07"
 
 # See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
 
 [dependencies]
-starknet = "2.4.0"
+starknet = "2.9.2"
```

Use `2.7.0` for now to align with the version [here](https://github.com/foundry-rs/starknet-foundry/blob/master/.tool-versions)
```suggestion
starknet = "2.7.0"
```